wicket-core/src/main/java/org/apache/wicket/core/request/handler/IPartialPageRequestHandler.java

```diff
@@ -78,6 +78,26 @@
 	 */
 	void appendJavaScript(CharSequence javascript);
 
+	/**
+	 * Add function for remote call execution on client side after components are replaced.
+	 * <p>
+	 * wicket-ajax-jquery will look for this function name inside of the Wicket.Ajax.RFC namespace
```

Comment:
How one register the function in this namespace ? This info would be very useful here.